Source/Editor/DualityEditor/DesignTimeObjectData.cs

```diff
@@ -184,6 +159,11 @@ public DesignTimeObjectData(Guid parentId, DesignTimeObjectData baseData)
 			this.attached = true;
 		}
 
+		public static DesignTimeObjectData Get(GameObject obj)
+		{
+			return DualityEditorApp.DualityEditorUserData.Instance.DesignTimeObjectDataManager.RequestDesignTimeData(obj.Id);
```

Comment:
This line (and the ones similar to it) is incredibly ugly 😄 Can we shorten that somehow? There's a ton of redundant information there. Suggestion:

`DualityEditorApp.UserData.Instance.PerObjectData.Request`

It involves three API method renames, potentially coupled with more of them to keep each classes API surface consistent with itself. I'd be fine with any subset of this as well. Reasoning:

- `UserData`: It's implicit that this is about editor user data, since it's a field defined in `DualityEditorApp` as opposed to `DualityApp`, with the return type backing this up when looking up the member.
- `PerObjectData`: Clarifies what this actually is instead of repeating the (not great) name of the return type.
- `Request`: The `DesignTimeObjectDataManager` class name already makes it clear what is requested here, so I think we can omit that similar to naming it `List<T>.Add` instead of `List<T>.AddItem`.